src/bdfs/Options.cpp

```diff
@@ -98,6 +98,26 @@ namespace dfs
     exit(format == NULL ? 0 : 1);
   }
 
+  uint64_t Options::ParseSizeStr(const std::string & ssize)
```

Comment:
We should still allow user to specify the value without the unit

Reply:
bdfs drive can read command line arg in bytes with parameter -s or can read UINT or String representation from conf json file.